ci-operator/step-registry/platform-external/pre/init/platform-external-pre-init-commands.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
```

Comment:
TODO review if we still need this step since those tools are available in `upi-installer` image. Are we using `upi-installer` across all required steps?